### PR TITLE
OLS-1290  - Adding waiting for pod running as a separate method

### PR DESCRIPTION
**Pull request by JoaoFula**
## Type of change

- [ x] Test fix

### CODE REVIEW

File: tests/e2e/utils/cluster.py [hunk 1]
```diff
@@ -294,3 +298,64 @@ def remove_file(pod_name: str, path: str) -> None:
...
+                    "--field-selector=status.phase=Pending",
+                    "-n",
+                    namespace,
+                ]
+            ).stdout
+        )
+        == 1,
+    )
+    r = retry_until_timeout_or_success(
+        120,
```
Comment (tisnik) [inline tests/e2e/utils/cluster.py:324]: OC_COMMAND_RETRY_COUNT ?

**Review by tisnik — CHANGES_REQUESTED** [review]
it looks ok, just it seems the named constant is not used

File: tests/e2e/utils/cluster.py (hunk 1 above)
Comment (JoaoFula) [reply tests/e2e/utils/cluster.py:324]: yes sir!

**Review by tisnik — APPROVED** [review]

**Comment by tisnik** [comment]
/override "Red Hat Konflux / ols-enterprise-contract / lightspeed-service"

**Pull request merged**